### PR TITLE
Rework Logs content

### PR DESCRIPTION
**Pull request by tamigoodall**
Moved and broke up files
Added new links

Review: https://docs-content-staging-pr-8604.herokuapp.com/docs/logs

### CODE REVIEW

File: articles/appliance/monitoring/index.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The PSaaS Appliance offers a number of tools to help you monitor the software th
...
 | Dashboards | The PSaaS Appliance's [Troubleshooting](/appliance/dashboard/troubleshoot) dashboard allows you to view [Health Check](/appliance/dashboard/troubleshoot#health-check) results for the past 29 days. Please note that this dashboard does not provide any alerts functionality and should not be used as your only monitoring strategy. |
-| Audit and Tenant Events | The [Tenant Logs](https://auth0.com/docs/logs) track the transactions the PSaaS Appliance processes and provide an overview of the activity occurring in your tenant by the applications dependent on your PSaaS Appliance. You can get a complete listing of events captured using the [log events manifest](/logs#log-data-event-listing). <br /> <br /> You can export your Tenant Logs to the third-party data tool of your choice using any of the [log export extensions](/extensions#export-auth0-logs-to-an-external-service) or the PSaaS Appliance's native log offloading functionality. <br /> <br /> By integrating with a third-party data/monitoring tool, such as Splunk, you can monitor and alert on activity that falls outside of bounds of normal functionality. |
```
Comment (erjohnson) [inline articles/appliance/monitoring/index.md:47]: Maybe this?
> The [Tenant Logs](https://auth0.com/docs/logs) track processed transactions and provide an overview of application activity in your tenant.

File: articles/logs/references/log-data-retention.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+To understand how you will use the Auth0 logs in your situation, review the log data retention period for your subscription level.
+
+Plan | Log Retention
+-----|--------------
+Free | 2 days
+Developer | 2 days
+Developer Pro | 10 days
+Enterprise | 30 days
+
+You can implement am Auth0 log data export extension to send log data to an external log analytics service. For example, you can use log files for troubleshooting and detecting intermittent errors that may be hard to find with quality assurance testing. You may also want log data in case forensic data is ever needed for security purposes. Log data can also provide comprehensive analytics to help you see patterns in usage trends and anomaly detection triggers.
```
Comment (erjohnson) [inline articles/logs/references/log-data-retention.md]: an*

File: articles/logs/references/log-data-retention.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+description: Describes how long log data is stored depending on your Auth0 plan.
+topics:
+  - logs
+  - log-data
+contentType:
+  - reference
+useCase:
+  - manage-logs
+---
+# Log Data Retention
```
Comment (erjohnson) [inline articles/logs/references/log-data-retention.md:12]:
```
# Log Data Retention

The retention period for your Auth0 logs is determined by your subscription level:

Plan | Log Retention
-----|--------------
Free | 2 days
Developer | 2 days
Developer Pro | 10 days
Enterprise | 30 days

For longer-term storage or analytics, you can export log data using a log export extension, rules, or the [AWS EventBridge Integration](/integrations/aws-eventbridge).
```

Comment (erjohnson) [reply articles/logs/references/log-data-retention.md:12]: Kinda shortening up the page, could add the example uses to the main index?

Comment (tamigoodall) [reply articles/logs/references/log-data-retention.md:12]: I'm going to keep them separate and add more crap from the Arch Scenarios here.

**Review by erjohnson — APPROVED** [review]

**Pull request merged**